Machine.c

```diff
@@ -54,7 +54,23 @@ void Machine_done(Machine* this) {
 #endif
 }
 
-void Machine_addList(Machine* this, struct ProcessList_ *pl) {
-   // currently only process lists are supported
-   this->pl = pl;
+void Machine_addTable(Machine* this, Table* table, bool processes) {
+   if (processes)
```

Comment:
The argument `processes` doesn't match the prototype declared in `Machine.h`

Reply:
Thanks, I've fixed this.